src/irr/asset/CMeshManipulator.cpp

```diff
@@ -1482,15 +1480,10 @@ bool CMeshManipulator::calcMaxQuantizationError(const SAttribTypeChoice& _srcTyp
         case EF_R16G16_SFLOAT:
         case EF_R16G16B16_SFLOAT:
         case EF_R16G16B16A16_SFLOAT:
```

Comment:
just remove these cases and let them drop to default.

Reply:
and remove the enums from here
https://github.com/buildaworldnet/IrrlichtBAW/blob/6b7aeccf2d8030799992b651fcd5546d10f79e48/src/irr/asset/CMeshManipulator.cpp#L1318

Reply:
this is still an issue

Reply:
still